contracts/DomainRegistry.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+import "hardhat/console.sol";
+import "./Ownable.sol";
+
+contract DomainRegistry is Ownable {
+    uint256 public totalDomains;
+    uint256 public totalReservedDomains;
```

Comment:
Consider making such vars `private` or `internal` if you already have public getters.